app/views/hello/index.html.erb

```diff
@@ -5,7 +5,7 @@
 <% @posts.each do |post| %>
   <div class="postmaster3000">
     <p><%= link_to post.user.full_name, user_path(post.user) %></p>
-    <%= image_tag(post.user.picture_url, class: "profile_pic") %>
+    <a href="<%= user_path post.user %>"><%= image_tag(post.user.picture_url, class: "profile_pic") %></a>
```

Comment:
this is great!

you can also use the `link_to` helper like we did on line 7. that "helper" will generate the `<a href"">` stuff for us injecting the url and the desired text into the right place.

```ruby
<%= link_to(user_path post.user) do %>
  <%= image_tag(post.user.picture_url, class: "profile_pic") %>
<% end %>
```

^ That will result in the same thing, but it's using a bit more Ruby and less manual HTML writing.